README.md

```diff
@@ -1 +1,143 @@
-# To-do-List
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+    <br/>
+
+  <h3><b>To do list README</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Awesome Books] <a name="about-project"></a>
+
+> Single page application that allows users to keep track of their tasks.
+
+**[To do List]** is a single page application that allows users to keep track of their tasks. Users can add the task and also check the checkbox if the task has been completed.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> HTML5
+> CSS
+> Javascript
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
```

Comment:
- Hi, Please, kindly remove this placeholder text as this text was added to guide you on updating your readme while using the template and should be removed after you've updated your readme.